index.html

```diff
@@ -1271,12 +1271,18 @@ <h2>Verification Methods</h2>
 described in Section [[[#verification-methods]]]. When it is used in either
 place, its purpose is essentially the same; that is, it expresses one or more
 entities that are authorized to perform certain actions associated with the
-resource with which it is associated.
+resource with which it is associated. Note however that for [=verification methods=] it is only
+an assertion made by the `controller` of the [=controlled identifier document=] and this 
+assertion may be true or false. It is not necessarily a statement of fact. 
+To ensure that a [=verification method=] 
+is bound to a particular [=controller=], one must go from the expression of the [=verification method=]
+to its [=controlled-identifier-document=], and then ensure that the latter contains a reference to 
+the [=verification method=] and to its [=controller=]. 
```

Comment:
```suggestion
resource with which it is associated.
Note however that for [=verification methods=], it is only
an assertion made by the `controller` of the [=controlled identifier document=] and this 
assertion may be true or false.
To ensure that a [=verification method=] 
is bound to a particular [=controller=], one must go from the expression of the [=verification method=]
to its [=controlled-identifier-document=], and then verify that the latter contains a reference to 
the [=verification method=] and to its [=controller=]. 
```

Tightened the language a bit.

Reply:
Working from @selfissued's suggestion... Note that the test from @David-Chadwick included both `[=controlled identifier document=]` and `[=controlled-identifier-document=]` (note the hyphens in the latter). It seems likely that only of these should be used, but I'm not sure which, offhand.
```suggestion
resource with which it is associated.
Note, however, that for [=verification methods=], it is only
an assertion made by the `controller` of the [=controlled identifier document=],
and this assertion is not necessarily true, i.e., it may be false.
To ensure that a [=verification method=] 
is bound to a particular [=controller=],
one must go from the expression of the [=verification method=]
to its [=controlled-identifier-document=],
and then verify that the latter contains references to both
the [=verification method=] and its [=controller=]. 
```